### PR TITLE
Feature/oui datagrid (uu-627)

### PR DESCRIPTION
**Pull request by whispyy**
datagrids :
- Comptes e-mail - Alias
- Groupes - Administrateurs
- Groupes - Alias
- Groupes - Membres

### CODE REVIEW

**Comment by FredericEspiau** [comment]
@whispyy could you please add the "Quality check" labels so we know if we can merge it or not ? :smiley:

File: src/exchange/account/alias/account-alias.controller.js [hunk 1]
```diff
@@ -11,36 +11,41 @@ angular
                 messaging,
                 translator
             };
-
             this.$routerParams = Exchange.getParams();
-            this.aliasLoading = false;
+            this.getAliasesParams = {};
```
Comment (JDownloader) [inline src/exchange/account/alias/account-alias.controller.js]: get should should be reserve as an instruction for function only, not for public variables

File: src/exchange/group/manager/group-manager.html [hunk 2]
```diff
@@ -6,53 +6,11 @@
...
-         data-ng-show="!ctrl.loading && ((ctrl.managersList.list.results | filter:{manager:true}).length > 0 || ctrl.managersList.list.messages.length > 0)"
-         data-pagination-server-side="managersTable"
-         data-pagination-server-side-function="getManagersByGroup"
-         data-pagination-server-side-paginated-stuff="getManagersList()"
-         data-pagination-server-side-table-loading="getLoading()">
-    </div>
+    <oui-datagrid data-rows-loader="ctrl.getManagersByGroup($config)">
+        <oui-column data-title="tr('exchange_tab_GROUPS_all_table_manager')" data-property="displayAddress"></oui-column>
+        <oui-action-menu data-align="end" data-compact>
+            <oui-action-menu-item data-text="{{i18n.exchange_tab_account_delete_account}}"
```
Comment (JDownloader) [inline src/exchange/group/manager/group-manager.html]: same tr(...) vs i18n.

File: src/exchange/group/member/group-member.html [hunk 3]
```diff
@@ -6,53 +6,11 @@
...
-         data-ng-show="!ctrl.loading && ((ctrl.membersList.list.results | filter:{member:true}).length > 0 || ctrl.membersList.list.messages.length > 0)"
-         data-pagination-server-side="membersTable"
-         data-pagination-server-side-function="getMembersByGroup"
-         data-pagination-server-side-paginated-stuff="getMembersList()"
-         data-pagination-server-side-table-loading="getLoading()">
-    </div>
+    <oui-datagrid data-rows-loader="ctrl.getMembersByGroup($config)">
+        <oui-column data-title="tr('exchange_tab_GROUPS_all_table_user')" data-property="displayAddress"></oui-column>
+        <oui-action-menu data-align="end" data-compact>
+            <oui-action-menu-item data-text="{{i18n.exchange_tab_account_delete_account}}"
```
Comment (JDownloader) [inline src/exchange/group/member/group-member.html]: same tr(...) vs i18n.

File: src/exchange/group/alias/group-alias.controller.js [hunk 4]
```diff
@@ -13,31 +13,43 @@ angular
...
-                    .catch((err) => {
-                        this.services.messaging.writeError(this.services.translator.tr("exchange_tab_ALIAS_error_message"), err);
-                    })
-                    .finally(() => {
-                        this.services.$scope.$broadcast("paginationServerSide.loadPage", 1, "groupAliasTable");
-                        this.loading = false;
-                    });
-            }
+            return this.services.Exchange
+                .getGroupAliasList(this.$routerParams.organization, this.$routerParams.productId, this.services.navigation.selectedGroup.mailingListAddress, pageSize, offset - 1)
```
[the remark this answers is not in the document]
Comment (whispyy) [reply src/exchange/group/alias/group-alias.controller.js]: I suppose it's some kind of some similar things we found in dedicated manager. That had a border effect using sidebar. But I don't really know if we're in that case in web!

File: src/exchange/account/alias/account-alias.controller.js [hunk 5]
```diff
@@ -11,36 +11,41 @@ angular
...
-                        this.services.messaging.writeError(this.services.translator.tr("exchange_tab_ALIAS_error_message"), err);
-                    })
-                    .finally(() => {
-                        this.aliasLoading = false;
-                    });
-            }
+        refreshList () {
+            this.services.Exchange.getAliases(this.$routerParams.organization, this.$routerParams.productId, this.services.ExchangeAccountService.selectedAccount.primaryEmailAddress, this.getAliasesParams.pageSize, this.getAliasesParams.offset - 1)
+                .then((data) => {
+                    for (let i = 0; i < data.list.results.length; i++) {
```
[the remark this answers is not in the document]
Comment (whispyy) [reply src/exchange/account/alias/account-alias.controller.js]: To explain a bit more about that.
Once there's an action affecting the array(add or remove). We need to refresh list for that we need to :
- retrieve the new array using the API
- check old and new array to drop or add changes

**Pull request merged**